website/ref/language.md

```diff
@@ -220,7 +220,7 @@ can be constructed by passing their **string representation** to the
 [`num`](builtin.html#num) builtin command:
 
 -   **Integers** are written in decimal (e.g. `10`), hexadecimal (e.g. `0xA`),
-    octal (e.g. `0o12`) or binary (e.g. `0b1010`).
+    octal (e.g. `012`, or `0o12`) or binary (e.g. `0b1010`).
```

Comment:
Fine by me although I would have swapped the two octal examples since, in some sense, the `0o` prefix is preferred and the `0` prefix exists for historical reasons. /pedantic /only-semi-serious 😄

Reply:
On a more serious note please change "Digits may be separately by underscores" to "Digits may be separated by underscores" a few lines farther down as part of this change. That is, change "separately" to "separated". That sentence needs to be rewritten but that is outside the scope of your change. Simply fixing "separately" to "separated" is sufficient and justifies an update to the P.R. that also changes the order of the octal examples per my previous comment. :smile:

Reply:
I'll also point out the examples should include at least one silly, yet legal, example  such as `1_2_3` being equivalent to `123`. Whether that improvement to the doc should be part of this change is debatable.